homework3/semi-uber/Server/index.js

```diff
@@ -14,8 +14,10 @@ mongoose.connect(dbURL, {
   useNewUrlParser: true
 });
 
+app.use(cors());
+
 app.use(express.json());
-app.use(allowCrossDomain);
+
 app.use("../api", express.static("api"));
```

Comment:
still have this quetsion. Will you use some static files in your application?

Reply:
At that moment - no